detection-rules/impersonation_coinbase.yml

```diff
@@ -9,9 +9,24 @@ severity: "high"
 source: |
   type.inbound
   and (
-    strings.ilike(sender.display_name, '*coinbase*')
+    any([
+          strings.replace_confusables(sender.display_name),
+          strings.replace_confusables(subject.subject)
```

Comment:
given the popularity of the brand within certain markets, the use of the word in the subject is not very strong.